vyxal/elements.py

```diff
@@ -5403,6 +5408,7 @@ def zfiller(lhs, rhs, ctx):
         "stack += [temp[-1]] + temp[:-1]",
         -1,
     ),
+    "🍪": process_element(cookie, 1),
```

Comment:
Wait why is the arity 1 here? You don't have any defined lhs in the function parameters so it doesn't make sense to do that.

Reply:
Well, there is a parameter taken, but it's `_`.

Reply:
> Wait why is the arity 1 here? You don't have any defined lhs in the function parameters so it doesn't make sense to do that.

Because I'm not quite sure how nilads that do nothing work

Reply:
Can't you just set arity 0 and only take `ctx` as an argument?

Reply:
Done